plugins/cache-portals-on-map.js

```diff
@@ -29,38 +29,37 @@ window.plugin.cachePortalDetailsOnMap.entityInject = function(data) {
   }
   data.callback(ents);
 };
-/****************************************************************************************/
-/* extend portal's GUID lookup to search in cached portals as well                      */
 
-window.plugin.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
-  let guid = window.plugin.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+// extend portal's GUID lookup to search in cached portals as well
+window.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
+  let guid = window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+
   // PLUGIN: cache > cache
   if (guid == null) {
     const cache = window.plugin.cachePortalDetailsOnMap;
     if (cache && cache.cache) {
       for (let g in cache.cache) {
         let p = cache.cache[g];
         if (latE6 == p.ent[2][2] && lngE6 == p.ent[2][3]) {
-          console.log("findPortalGuidByCacheE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
+//        console.log("newFindPortalGuidByPositionE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
           guid = g;
           break;
         }
       }
     }
   }
   return guid;
-}
+};
+
+window.cachePortalDetailsOnMap.searchInit = function () {
+    window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old = window.findPortalGuidByPositionE6;
+    window.findPortalGuidByPositionE6 = window.cachePortalDetailsOnMap.findPortalGuidByCacheE6;
+};
```

Comment:
Why do we ever need separate function for 2-lines code?

Reply:
What is the problem?
There are way more functions that only consist of ONE line.
But I see that most plugins have their init-routines separated from setup().

Reply:
> What is the problem?

It's hard to read such spaghetti code. 

> There are way more functions that only consist of ONE line.
> But I see that most plugins have their init-routines separated from setup().

You see legacy code, so no surprise that there are a lot of bad patterns.

You should not do like this, in general.